### PR TITLE
Align RBAC on Microsoft.Network

### PR DESCRIPTION
**Pull request by MariusStorhaug**
# Change

- Moved `builtInRoleNames` variable to `.bicep/nested_rbac.json`
- Using declaration of existing resource + scope reference in roleAssignment resource declaration
- Updated Readme

## Type of Change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update (Wiki)

## Checklist

- [x] I'm sure there are no other open Pull Requests for the same update/change
- [ ] My corresponding pipelines / checks run clean and green without any errors or warnings
- [x] My code follows the style guidelines of this project
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation (readme)
- [x] I did format my code

### CODE REVIEW

File: arm/Microsoft.Network/expressRouteCircuits/readme.md [hunk 1]
```diff
@@ -7,15 +7,15 @@ This template deploys a ExrepressRoute Circuit.
...
 | `Microsoft.Insights/diagnosticSettings` | 2017-05-01-preview |
 | `Microsoft.Network/expressRouteCircuits` | 2021-02-01 |
-| `Microsoft.Network/expressRouteCircuits/providers/roleAssignments` | 2021-04-01-preview |
 
 ## Parameters
 
 | Parameter Name | Type | Default Value | Possible Values | Description |
 | :-- | :-- | :-- | :-- | :-- |
-| `bandwidthInMbps` | int |  |  | Required. This is the bandwidth in Mbps of the circuit being created. It must exactly match one of the available bandwidth offers List ExpressRoute Service Providers API call. SkuTier 'Local' has special bandwith requirements. It requires minimum 1000mbps.|
+| `bandwidthInMbps` | int |  |  | Required. This is the bandwidth in Mbps of the circuit being created. It must exactly match one of the available bandwidth offers List ExpressRoute Service Providers API call. |
```
Comment (MrMCake) [inline arm/Microsoft.Network/expressRouteCircuits/readme.md:18]: Outside of scope but still: Any chance you could add a `@minValue(1000)` to the `bandwidthInMbps` in the _deploy.bicep_ file?

Comment (MariusStorhaug) [reply arm/Microsoft.Network/expressRouteCircuits/readme.md:18]: I can, but it says `SkuTier 'Local' has special bandwidth requirements. It requires minimum 1000mbps`. Are these two sentences linked? Is the limit conditional?

Comment (MrMCake) [reply arm/Microsoft.Network/expressRouteCircuits/readme.md:18]: Just checked the template reference. Doesn't seem like this sentence exists there. Beggs the question if that limitation actually exists

Comment (MrMCake) [reply arm/Microsoft.Network/expressRouteCircuits/readme.md:18]: Nope it does not. Please ignore

**Review by MrMCake — APPROVED** [review]

**Comment by phorn30** [comment]
This likely fixes the issue I raised in issue #286.  The resource deployment worked but the RBAC application failed.

**Pull request merged**